swap/swap.go

```diff
@@ -561,13 +591,21 @@ func (s *Swap) saveBalance(p enode.ID, balance int64) error {
 	return s.store.Put(balanceKey(p), balance)
 }
 
+func loadPaidout(s state.Store) (paidOut uint64, err error) {
+	err = s.Get(paidoutKey, &paidOut)
+	if err == state.ErrNotFound {
+		return 0, nil
+	}
+	return paidOut, err
+}
+
 // Close cleans up swap
 func (s *Swap) Close() error {
 	return s.store.Close()
 }
 
 // GetParams returns contract parameters (Bin, ABI, contractAddress) from the contract
-func (s *Swap) GetParams() *swap.Params {
+func (s *Swap) GetParams() *contract.Params {
```

Comment:
not clear on this one

Reply:
We imported twice the `contracts/swap` package. Once with an alias `contract` and once just without the alias. 
I reverted this, and now only the import with the alias (`contract`) is there.